sigstore/_internal/ctfe.py

```diff
@@ -25,7 +25,7 @@
 from cryptography.hazmat.primitives import hashes
 from cryptography.hazmat.primitives.asymmetric import ec, rsa
 
-from sigstore._utils import key_id, load_pem_public_key
+from sigstore._utils import key_id, keyid, load_pem_public_key
```

Comment:
Hmm, having both `key_id` (a function) and `keyid` (a type) is definitely a little confusing. 

Let's go with CamelCase for these types after all:

* `keyid -> KeyID`
* `hexstr -> HexStr`
* `b64str -> B64Str`
* `pemcert -> PEMCert`
* `dercert -> DERCert`